### DISCUSSION TITLE
According to the postcss cli https://github.com/postcss/postcss-cli, there is no <output-glob-pattern> is there any workaround?

### DISCUSSION THREAD

**Discussion by aaacafe786** [postcss/postcss#1745] [General]
hello there, we really wanted to use postcss with autoprefixer in our remix/scss project, the issue is that according to the postcss cli https://github.com/postcss/postcss-cli, there is no <output-glob-pattern> is there any workaround?

**Comment by ai** [comment]
You can write a simple Node.js script using PostCSS JS API to do the `postcss-cli` work.

It will be a 10-20 LOC script.

**Reply by aaacafe786** [reply]
thank you so much for the swift response, do you have any relevant documentation or similar examples? 

Kind Regards
Alym Karim

**Reply by ai** [reply]
```js
let postcss = require('postcss')
let autoprefixer = require('autoprefixer')

let result = await postcss([autoprefixer]).process(css, { from: filePathFrom, to: filePathTo })
for (let warning of result.warnings()) {
  console.warn(warning)
}
result.css //=> output CSS
```

The API: https://postcss.org/api/

**Comment by aaacafe786** [comment]
is there any way to convert scss/scss-like-css to css in the same step?

**Reply by ai** [reply]
SCSS should have JS API too, but I can’t help here.
